jenkins.check_copyright.py

```diff
@@ -30,7 +30,7 @@
     exit(0)
 
 with open(file_name, 'r') as sourcefile:
-    for x in range(0,len(copyright_lines)):
+    for regex in regexes:
         # TODO add a check for empty files (dummy.js etc), as they cause the script to crash here
         line = next(sourcefile)
         line = line.rstrip()
```

Comment:
Got it! I see my mistake, thank you. I already corrected the error and commited the change. Is there anything else I should do?